{"name": "test_organization_different_from_idiomatic", "status": "failed", "statusDetails": {"message": "AssertionError: Unexpected code: 500", "trace": "admin_token = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJkYXRhIjp7ImVuZHVzZXIuYXV0aC5sb2dpblRpbWUiOiIxNjQ4NjM0NDY3IiwiZW5kdXNlci5hdXRo...ZXIuYXV0aC5lbmRVc2VySWQiOiI4NDk3In0sImlhdCI6MTY0ODYzNCwibmJmIjoxNjQ4NjM0fQ.LyQ4o9gMZlGdkhaByY16rTSajPNGUtLPSbVEGc89Ab8'\n\n    def test_organization_different_from_idiomatic(admin_token):\n>       PredictTicketText(admin_token).request().verify_response(error_code=400)\n\ntests/enduser/test_predict_ticket_text.py:48: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <qa.api.enduser.calls.PredictTicketText object at 0x1042b08b0>, error_code = 400, resp_body = None\nignore_order = False, exclude_paths = None, normalize = False\n\n    def verify_response(self, error_code=None, resp_body=None,\n                        ignore_order=False, exclude_paths=None,\n                        normalize=False):\n        if error_code:\n>           assert self.response.status_code == error_code, (\n                f'Unexpected code: {self.response.status_code}'\n            )\nE           AssertionError: Unexpected code: 500\n\nqa/tools/caller.py:83: AssertionError"}, "start": 1648640667262, "stop": 1648640667652, "uuid": "aedc2519-8daa-4d6c-8723-0a4699bfd6c5", "historyId": "b5bc92a5bfd4ef08dd714a20ae77492a", "testCaseId": "0a0f0df5558b8b8c2b1f121ab67de1d8", "fullName": "tests.enduser.test_predict_ticket_text#test_organization_different_from_idiomatic", "labels": [{"name": "parentSuite", "value": "tests.enduser"}, {"name": "suite", "value": "test_predict_ticket_text"}, {"name": "host", "value": "Elenas-MacBook-Pro.local"}, {"name": "thread", "value": "59719-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.enduser.test_predict_ticket_text"}]}